packages/cli/src/checks.ts

```diff
@@ -72,16 +63,24 @@ export const runChecks = async (
   );
 
   const total = rulesToRunAccordingToLevel.length + 1;
+  let current = 0;
 
-  const rulesProgressBar = progressBar.create(
-    total,
-    0,
-    {},
-    { format: 'Rules:  {bar} {percentage}% | ETA: {eta}s | {value}/{total}' },
-  );
+  const rulesSpinner = new Spinner({
+    text: 'Processing rules... %s',
+    stream: process.stderr,
+
+    onTick: function (msg) {
+      this.clearLine(this.stream);
+      this.stream.write(msg);
+    },
+  });
+  rulesSpinner.setSpinnerString('◢◣◤◥');
+
+  rulesSpinner.start();
 
   const decreaseRemaining = () => {
-    rulesProgressBar.increment();
+    current++;
+    rulesSpinner.setSpinnerTitle(`Processing rule ${current}/${total}... %s`);
```

Comment:
I would put the %s directly at the start, in order to avoid confusion with the cursor